runelite-client/src/main/java/net/runelite/client/plugins/statusbars/BarRenderer.java

```diff
@@ -0,0 +1,170 @@
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ *
+ */
+
+package net.runelite.client.plugins.statusbars;
+
+import lombok.RequiredArgsConstructor;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.BACKGROUND;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.COUNTER_ICON_HEIGHT;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.HEAL_OFFSET;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.ICON_AND_COUNTER_OFFSET_X;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.ICON_AND_COUNTER_OFFSET_Y;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.OFFSET;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.OVERHEAL_COLOR;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.OVERHEAL_OFFSET;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.PADDING;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.SKILL_ICON_HEIGHT;
+import static net.runelite.client.plugins.statusbars.StatusBarsOverlay.WIDTH;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.components.TextComponent;
+
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.Image;
+import java.util.function.Supplier;
+
+@RequiredArgsConstructor
+class BarRenderer
+{
+	private static final TextComponent textComponent = new TextComponent();
+
+	private final StatusBarsConfig config;
+	private final Supplier<Integer> maxValueSupplier;
+	private final Supplier<Integer> currentValueSupplier;
+	private final Supplier<Integer> healSupplier;
+	private final Supplier<Color> colorSupplier;
+	private final Supplier<Color> healColorSupplier;
+	private final Supplier<Image> iconSupplier;
+
+	private int maxValue;
+	private int currentValue;
+
+	private void refreshSkills()
+	{
+		maxValue = maxValueSupplier.get();
+		currentValue = currentValueSupplier.get();
+	}
+
+	private void renderBar(Graphics2D graphics, int x, int y, int height)
+	{
+		final Color fill = colorSupplier.get();
+
+		graphics.setColor(BACKGROUND);
+		graphics.drawRect(x, y, WIDTH - PADDING, height - PADDING);
+		graphics.fillRect(x, y, WIDTH, height);
+
+		final int filledHeight = getBarHeight(maxValue, currentValue, height);
+		graphics.setColor(fill);
+		graphics.fillRect(x + PADDING,
+				y + PADDING + (height - filledHeight),
```

Comment:
Just use a single indent here.
```suggestion
			y + PADDING + (height - filledHeight),
```